packages/compose/src/hooks/use-focus-return/index.js

```diff
@@ -45,13 +47,15 @@ function useFocusReturn( onFocusReturn ) {
 				return;
 			}
 
-			focusedBeforeMount.current = node.ownerDocument.activeElement;
+			focusedBeforeMount.current =
+				/** @type {HTMLElement | null} */ ( node.ownerDocument
+					.activeElement ) || undefined;
```

Comment:
Is there a reason to prefer `|| undefined` here than `?? undefined`? Also, why prefer `undefined` than potentially `null` here?

Reply:
None, I can change it to `??` just fine.

I used `undefined` because it matches the original default value on line 34. We could change it to `null` if we change the default value to `null` as well. Do you think that would be better?

Reply:
Thanks, I like that 👍